service/controller/resource/promremotewrite/create.go

```diff
@@ -0,0 +1,75 @@
+package promremotewrite
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/v7/pkg/controller/context/resourcecanceledcontext"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	r.logger.Debugf(ctx, "reconcile remotewrite")
+	{
+		// get remotewrite
+		remoteWrite, err := ToRemoteWrite(obj)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		selector, err := metav1.LabelSelectorAsSelector(&remoteWrite.Spec.ClusterSelector)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		// fetch current prometheus
+		prometheusList, err := r.prometheusClient.
+			MonitoringV1().
+			Prometheuses(metav1.NamespaceAll).
+			List(ctx, metav1.ListOptions{LabelSelector: selector.String()})
+		if err != nil {
+			return microerror.Maskf(errorFetchingPrometheus, "Could not fetch Prometheus with label selector '%#q'", remoteWrite.Spec.ClusterSelector.String())
```

Comment:
Quotes are already added when using `%q`
```suggestion
			return microerror.Maskf(errorFetchingPrometheus, "Could not fetch Prometheus with label selector %#q", remoteWrite.Spec.ClusterSelector.String())
```